tests/ut_textchannellistener/ut_textchannellistener.cpp

```diff
@@ -129,8 +129,11 @@ void Ut_TextChannelListener::cleanupTestCase()
         model.getGroups(IM_ACCOUNT_PATH);
         QVERIFY(waitSignal(modelReady, 5000));
 
-        while (model.rowCount() != 0)
+        while (model.rowCount() != 0) {
+            int waitForDelete = model.rowCount()-1;
             QVERIFY(model.deleteGroups(QList<int>() << model.group(model.index(0,0)).id()));
+            QTRY_COMPARE(model.rowCount(), waitForDelete);
```

Comment:
Not really that familiar with the details here, but since GroupModel::deleteGroups() is defined to return true if deletion was succesful, how can rowCount change with delay? By APIs I'd expect QCOMPARE() to pass already.